src/esp/sim/Simulator.cpp

```diff
@@ -477,6 +486,52 @@ bool Simulator::instanceObjectsForActiveScene() {
   return true;
 }  // Simulator::instanceObjectsForActiveScene()
 
+bool Simulator::instanceArticulatedObjectsForActiveScene() {
+  // Get scene instance attributes corresponding to current active scene name
+  // This should always just retrieve an existing, appropriately configured
+  // scene instance attributes, depending on what exists in the Scene Dataset
+  // library for the current dataset.
+  const std::string activeSceneName = config_.activeSceneName;
+  metadata::attributes::SceneAttributes::cptr curSceneInstanceAttributes =
+      metadataMediator_->getSceneAttributesByName(activeSceneName);
+
+  // get lightSetupKey from the value set when stage was created.
+  const std::string lightSetupKey = config_.sceneLightSetup;
```

Comment:
This appears to be unused. See also my earlier comment about URDF-loading using a hard-coded light setup key.

Reply:
Resolved. This is now passed through, so AO's should be lit the same as other objects in the scene.